abc.templates/base-workflows/contents/guardian-plan.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           SOURCE_REF: '${{ github.event.pull_request.base.sha }}'
           DEST_REF: '${{ github.event.pull_request.head.sha }}'
         run: |-
-          DIRECTORIES=$(guardian entrypoints -detect-changes -source-ref="${SOURCE_REF}" -dest-ref="${DEST_REF}" -format=json REPLACE_TERRAFORM_DIRECTORY)
+          DIRECTORIES=$(guardian entrypoints -dir=REPLACE_TERRAFORM_DIRECTORY -detect-changes -source-ref="${SOURCE_REF}" -dest-ref="${DEST_REF}" -format=json)
```

Comment:
nit: can we wrap the dir in double quotes?

Reply:
done